backends/bmv2/common/helpers.h

```diff
@@ -66,11 +56,204 @@ class V1ModelProperties {
     static const cstring validField;
 };
 
+namespace Standard {
```

Comment:
Code below was copied from control-plane/p4RuntimeArchStandard.cpp.

Reply:
Do we need it to be copied, or can we share it somehow?

Reply:
The `mapUnitName` method in CounterlikeTraits depends on protobuf, which does not make much sense to carried over to bmv2 backend. See line 190 in p4RuntimeArchStandard.cpp. @antoninbas FYI.